cmd/authy-export/authy-export.go

```diff
@@ -12,6 +12,7 @@ import (
 	"strconv"
 	"strings"
 	"time"
+	"flag"
```

Comment:
This `flag` import would get reordered to be just below `errors`, if you ran `go fmt` over this code.

Please run `go fmt` over the package for the next round, thanks! You can usually get your editor to do this for you automatically.